ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js

```diff
@@ -33,22 +33,22 @@ describe("Form in site using embed container", () => {
         expect(formContainer.length).to.equal(2);
       });
 
-      it.skip("should submit on button click and thank you message of embed container must be overridden", () => {
+      it("should submit on button click and thank you message of embed container must be overridden", () => {
```

Comment:
Both these test should only run in latestAddon. can you add the cy.af.isLatestAddon check ? you can check for the occurrences

Reply:
Added